R/dummy_functions.R

```diff
@@ -53,34 +53,49 @@ get_dummy_filter <- function(data) { # nolint
 #' @return `cdisc_data`
 #' @keywords internal
 get_dummy_cdisc_data <- function() { # nolint
-  teal_with_pkg <- function(pkg, code) {
-    pkg_name <- paste0("package:", pkg)
-    if (!pkg_name %in% search()) {
-      require(pkg, character.only = TRUE)
-      on.exit(detach(pkg_name, character.only = TRUE))
-    }
-    eval.parent(code)
-    return(invisible(NULL))
-  }
-
-  teal_with_pkg("scda", code = {
-    ADSL <- scda::synthetic_cdisc_data("latest")$adsl # nolint
-    ADAE <- scda::synthetic_cdisc_data("latest")$adae # nolint
-    ADLB <- scda::synthetic_cdisc_data("latest")$adlb # nolint
-  })
+  ADSL <- data.frame( # nolint
+    STUDYID = "study",
+    USUBJID = 1:10,
+    SEX = sample(c("F", "M"), 10, replace = TRUE),
+    AGE = stats::rpois(10, 40)
+  )
+  ADTTE <- rbind(ADSL, ADSL, ADSL) # nolint
+  ADTTE$PARAMCD <- rep(c("OS", "EFS", "PFS"), each = 10) # nolint
+  ADTTE$AVAL <- c( # nolint
+    stats::rnorm(10, mean = 700, sd = 200), # dummy OS level
+    stats::rnorm(10, mean = 400, sd = 100), # dummy EFS level
+    stats::rnorm(10, mean = 450, sd = 200) # dummy PFS level
+  )
 
   ADSL$logical_test <- sample(c(TRUE, FALSE, NA), size = nrow(ADSL), replace = TRUE) # nolint
-  ADSL$SEX[1:150] <- NA # nolint
+  ADSL$SEX[c(2, 5)] <- NA # nolint
+
+  cdisc_data_obj <- cdisc_data(
```

Comment:
```suggestion
  cdisc_data_obj <- teal.data::cdisc_data(
```

Reply:
Even though `teal.data` is in `Depends`?